### PR TITLE
DOC: Extend the demo

### PR DESCRIPTION
**Pull request by hanke**
Detected issues:

- `publish` doesn't use ssh connection manager properly #544

### CODE REVIEW

File: docs/examples/3rdparty_analysis_workflow.sh [hunk 1]
```diff
@@ -89,6 +89,12 @@ datalad update alice --merge
 datalad install result.txt
```
Comment (yarikoptic) [inline docs/examples/3rdparty_analysis_workflow.sh]: should there be some commit message, so it looks like a lab book, e.g.

datalad install --msg "Opened the black hole, TODO: find the close toggle" result.txt

Comment (hanke) [inline docs/examples/3rdparty_analysis_workflow.sh]: Related to #546

Comment (hanke) [inline docs/examples/3rdparty_analysis_workflow.sh]: At present there would be no commit, only an `add` of that file.

Comment (yarikoptic) [inline docs/examples/3rdparty_analysis_workflow.sh]: ah -- so there is a missing "save" call which should come before any publish?

Comment (hanke) [inline docs/examples/3rdparty_analysis_workflow.sh]: No, the line you referenced obtains the change made by alice in bob's dataset, hence all is ready for publish and apart from data availability there is no change.

Comment (yarikoptic) [inline docs/examples/3rdparty_analysis_workflow.sh]: ah, I have missed that result.txt is under annex not under git.

**Pull request merged**